### PR TITLE
add php unittest support

### PR DESCRIPTION
**Pull request by dothebart**
add php test.

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/11915/

**Comment by dothebart** [comment]
test green.

File: js/client/modules/@arangodb/testsuites/drivers.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+// / Copyright holder is ArangoDB GmbH, Cologne, Germany
+// /
+// / @author Wilfried Goesgens
+// //////////////////////////////////////////////////////////////////////////////
+
+const functionsDocumentation = {
+  'driver': 'generic driver test - requires driver adaptor script',
+};
+const optionsDocumentation = [
+  '   - `driversource`: directory of the java driver',
```
Comment (jsteemann) [inline js/client/modules/@arangodb/testsuites/drivers.js:33]:
```suggestion
  '   - `driversource`: directory of the arangodb-php driver',
```

Comment (jsteemann) [reply js/client/modules/@arangodb/testsuites/drivers.js:33]: Or what kind of driver is expected here?

**Review by jsteemann — APPROVED** [review]

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/11927/

**Pull request merged**